packages/front-end/components/Experiment/ChanceToWinColumn.tsx

```diff
@@ -76,6 +77,22 @@ export default function ChanceToWinColumn({
         className
       )}
     >
+      {suspiciousChange && (
```

Comment:
Do we also need to check `enoughData` here?  If one variation has 3 conversions and another has 1, that's a 300% increase, but we shouldn't show the suspicious badge since it's not enough data to draw any conclusions yet.

Reply:
Ah, yes - thank you for catching that. Updated.